docs/source/techniques/ISISPowder-Polaris-v1.rst

```diff
@@ -167,21 +168,33 @@ create_total_scattering_pdf
              state should not be considered accurate or complete.
 
 The *create_total_scattering_pdf* method allows a user to create a Pair Distribution Function (PDF)
-from focused POLARIS data, with a view performing further total scattering analysis.
+from focused POLARIS data, with a view to performing further total scattering analysis.
 
-This function applies the placzek self scattering correction from
-:ref:`algm-CalculatePlaczekSelfScattering` before calculating the PDF
+This function takes as input a workspace group containing focussed data that has been normalized as a differential cross section. The input can be generated by running
+:ref:`focus_polaris_isis-powder-diffraction-ref` with van_normalisation_method=Absolute.
 
-This function requires the run_number you wish to analyse. The focused file for this run number must
+The function performs the following processing:
+
+- converts the x units of the workspace group containing the differential cross section to momentum transfer
+- calculates the Placzek self scattering algorithm using the workflow algorithm :ref:`algm-TotScatCalculateSelfScattering`.
+  This internally calculates a per detector Placzek correction using :ref:`algm-CalculatePlaczekSelfScattering-v1` and then focusses the correction using :ref:`algm-GroupDetectors`
+- subtracts the Placzek self scattering correction from the differential cross section
+- converts the differential cross-section into an :math:`S(Q) - 1` distribution using the following relationship where :math:`\sigma_s` is the total scattering cross section and :math:`\sigma_{coh}` is the coherent scattering cross section. This is based on equations 5, 9 and 19 in [#Keen]_:
+
+:math:`\hspace{20pt}S(Q) - 1 = (\frac{d\sigma_s}{d\Omega}(Q) - <b_{tot}^2>) / <b_{coh}>^2 = (\frac{d\sigma_s}{d\Omega}(Q) - \frac{\sigma_s}{4 \pi}) / <b_{coh}>^2`
```

Comment:
I've just pushed a commit and I went for your suggested change in the end (rather than changing the code to match the documentation). This was partly because it seems cleaner having the equation all in terms of cross sections instead of a mix of one cross section and one scattering length. Also the cross sections are visible to the user in the UI on the right click "Show Sample Material" menu on a workspace so perhaps these quantities are a bit more user friendly:

![image](https://user-images.githubusercontent.com/56295817/181452835-764ed240-2d4c-4e1b-96a3-7e949311dfe2.png)